deploy/create-export-url-bucket.sh

```diff
@@ -0,0 +1,33 @@
+  #!/bin/bash -xb
+#
+# Creates GCS bucket for Export to Saturn feature.
+#
+# See export_url_controller.py for more information.
+#
+# jq, gsutil must be installed before running this script.
+
+if (( $# != 1 ))
+then
+  echo "Usage: deploy/create-export-url-bucket.sh <dataset>"
+  echo "  where <dataset> is the name of a directory in dataset_config/"
+  echo "Run this script from project root"
+  exit 1
+fi
+
+dataset=$1
+project_id=$(jq --raw-output '.project_id' dataset_config/${dataset}/deploy.json)
```

Comment:
Would it be easier to just make this a python script so you get things like json parsing for free? The cloud storage python API seems to support all the things you need for bucket management: https://googlecloudplatform.github.io/google-cloud-python/latest/storage/buckets.html

Reply:
Shell was easier to write